parcel_test.go

```diff
@@ -31,58 +33,139 @@ func getTestParcel() Parcel {
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
 	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "tracker.db") // настройте подключение к БД
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	defer func(db *sql.DB) {
+		_ = db.Close()
```

Comment:
Если ошибка все равно не обрабатывается, то можно не делать отдельную функцию
```suggestion
		defer db.Close()
```

Reply:
Здесь и далее